src/join-9.sol

```diff
@@ -18,7 +18,7 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-pragma solidity >=0.6.12;
+pragma solidity ^0.6.12;
```

Comment:
yeah, it diverged a bit from current join pragma patterns, but in the light of recent `dss` housekeeping, locking to solc major version seems to be the best practise here, to avoid the case of compiling with `0.7.x` and higher that could break the code as breaking changes are usually introduced in major solc releases.

Reply:
Let's add that to the discussion.

Reply:
I'd suggest moving `pragma solidity >= 0.5.12;` ones to `pragma solidity ^0.5.12 || ^0.6.12;` and the rest as `pragma solidity ^0.6.12;`

Reply:
Hmm, not sure about the pragma statement suggested, as there should be breaking changes introduced with `0.6.12`, so for backward compatibility this could break compilation.

Reply:
I say we keep some sort of standardization as I wrote before. Or we set it back to >=0.6.12 or we need to change everywhere. For the seek of simplification, please roll this back.

Reply:
Once this one is rolled back I'm gonna give my approval.

Reply:
done in b492377